components/event-sources/adapter/http/README.md

```diff
@@ -0,0 +1,70 @@
+# HTTP Adapter Source
+
+The http adapter is a http server which receives [CloudEvents](https://github.com/cloudevents/spec) in version 1.0 and proxies them to a preconfigured sink.
+The adapter is written with the [cloudevents sdk-go](https://github.com/cloudevents/sdk-go) and is fully compatible with CloudEvents specification.
+
+## Specification
+
+The http adapter is CloudEvents 1.0 compatible. It accepts binary as well as structured content mode. Only batched mode is not implemented.
+Read more about the content types and the http protocol binding [here](https://github.com/cloudevents/spec/blob/master/http-protocol-binding.md#13-content-modes).
+
+## Run
+
+The adapter can be run locally.
+
+It requires a few variables to be set:
+
+```bash
+export SINK_URI=http://localhost:55555
+export NAMESPACE=foo
+export K_METRICS_CONFIG='{"config-observability":"{\"metrics.backend\":\"prometheus\"}"}'
+export K_LOGGING_CONFIG='{"zap-logger-config": "{\"level\":\"info\",\"development\":\"true\",\"outputPaths\":[\"stdout\"],\"errorOutputPaths\":[\"stderr\"],\"encoding\":\"console\",\"encoderConfig\":{\"timeKey\":\"ts\",\"levelKey\":\"level\",\"nameKey\":\"logger\",\"callerKey\":\"caller\",\"messageKey\":\"msg\",\"stack traceKey\":\"stacktrace\",\"lineEnding\":\"\",\"levelEncoder\":\"\",\"timeEncoder\":\"iso8601\",\"durationEncoder\":\"\",\"callerEncoder\":\"\"}}"}'
+export EVENT_SOURCE="varkes"
+```
+
+Run it:
+```bash
+go run cmd/http-adapter/main.go
+```
+
+The adapter will send events to the `SINK_URI`. When running the adapter locally, you can simple use `netcat` as a sink:
+```bash
+printf "HTTP/1.1 200 OK\r\n\r\n" | nc -vl 55555
+```
+
+Send an event to the adapter:
+
+```bash
```

Comment:
(nit) `console` provides a better highlighting for console commands (anything starting with `#`, `$` followed by a command and then some output)

Reply:
I separated the commands into two bash blocks since the curl output is never shown I did not use `console`. I also removed the leading `$`. Please check if that is ok for you :)